src/apps/basket/tests.py

```diff
@@ -0,0 +1,81 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from src.apps.basket.basket import Basket
+from src.apps.inventory.models import Product, Category
+
+
+class BasketAddViewTest(TestCase):
+    def setUp(self):
+        self.category = Category.objects.create(name="Test Category")
+        self.user_seller = get_user_model().objects.create_user(
+            username="seller", password="password", user_type="Seller"
+        )
+        self.product = Product.objects.create(
+            name="Test Product",
+            category=self.category,
+            price=10.0,
+            seller=self.user_seller,
+            stock=10,
+        )
+
+    def test_add_product_to_basket(self):
```

Comment:
Consider giving the test cases more descriptive names that clearly explain what scenario they are testing.
e.g. test_add_product_to_basket_with_sufficient_stock

Reply:
I believe it could be redundant